.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 100,
+  "trailingComma": "all",
+  "singleQuote": true
```

Comment:
put some defaults that I use personally. the only difference I think was the `trailingComma` option. what do you prefer?

Reply:
Trailing makes sense